src/main/java/graphql/util/Anonymizer.java

```diff
@@ -785,40 +826,30 @@ public TraversalControl visitInlineFragment(InlineFragment node, TraverserContex
+                String newName = assertNotNull(astNodeToNewName.get(node));
                 return changeNode(context, node.transform(builder -> builder.name(newName)));
             }
 
             @Override
-            public TraversalControl visitArgument(Argument node, TraverserContext<Node> context) {
-                String newName = assertNotNull(nodeToNewName.get(node));
-                return changeNode(context, node.transform(builder -> builder.name(newName)));
+            public TraversalControl visitArgument(Argument argument, TraverserContext<Node> context) {
+                GraphQLArgument graphQLArgumentDefinition;
+                // An argument is either from a applied query directive or from a field
+                if (context.getVarFromParents(GraphQLDirective.class) != null) {
+                    GraphQLDirective directiveDefinition = context.getVarFromParents(GraphQLDirective.class);
+                    graphQLArgumentDefinition = directiveDefinition.getArgument(argument.getName());
+                } else {
+                    GraphQLFieldDefinition graphQLFieldDefinition = assertNotNull(context.getVarFromParents(GraphQLFieldDefinition.class));
+                    graphQLArgumentDefinition = graphQLFieldDefinition.getArgument(argument.getName());
+                }
+                GraphQLInputType argumentType = graphQLArgumentDefinition.getType();
+                String newName = assertNotNull(astNodeToNewName.get(argument));
+                Value newValue = replaceValue(argument.getValue(), argumentType, newNames, defaultStringValueCounter, defaultIntValueCounter);
+                return changeNode(context, argument.transform(builder -> builder.name(newName).value(newValue)));
             }
         });
         return AstPrinter.printAstCompact(newDocument);
     }
 
-//    private void findAllTheSameFields(GraphQLSchema schema) {
-//        Map<GraphQLFieldDefinition, Collection<GraphQLFieldDefinition>> sameFields = new LinkedHashMap<>();
-//
-//        GraphQLTypeVisitor visitor = new GraphQLTypeVisitorStub() {
-//            @Override
-//            public TraversalControl visitGraphQLFieldDefinition(GraphQLFieldDefinition graphQLFieldDefinition, TraverserContext<GraphQLSchemaElement> context) {
-//                String curName = graphQLFieldDefinition.getName();
-//                GraphQLImplementingType parentNode = (GraphQLImplementingType) context.getParentNode();
-//                List<GraphQLNamedOutputType> interfaces = parentNode.getInterfaces();
-//                List<GraphQLFieldDefinition> matchingInterfaceFieldDefinitions = getMatchingInterfaceFieldDefinitions(curName, interfaces);
-//                if (matchingInterfaceFieldDefinitions.size() > 0) {
-//                    sameFields.put(graphQLFieldDefinition, matchingInterfaceFieldDefinitions);
-//                }
-//            }
-//
-//        };
-//
-//        SchemaTransformer.transformSchema(schema, visitor);
-//
-//    }
-
```

Comment:
commented out code is deleted code right

Reply:
Yeah :) ... will clean it up. I overlooked it